.github/workflows/daily.yml

```diff
@@ -0,0 +1,33 @@
+name: "Daily tasks (cronjobs)"
```

Comment:
Can you add Robotoff in the job name, as well as a clarification on what it does (a backup)